pkg/ctl/create/cluster.go

```diff
@@ -32,7 +32,8 @@ var (
 )
 
 func createClusterCmd() *cobra.Command {
-	cfg := &api.ClusterConfig{}
+	cfg := api.NewClusterConfig()
+	ng := &cfg.NodeGroups[0]
```

Comment:
This is not idea, but what would be better? `cfg.GetDefaultNodeGroup()` or perhaps `cfg, ng := api.NewClusterConfig()`?

Reply:
@Lazyshot you are right, let's make it a distinct step!